packages/client/hooks/useMutationProps.tsx

```diff
@@ -59,7 +59,7 @@ const useMutationProps = () => {
     setSubmitting(true)
   }, [])
 
-  return {submitting, submitMutation, error, onError, onCompleted}
+  return {submitting, submitMutation, error, setError, onError, onCompleted}
```

Comment:
-1 why doesn't `onError` work? I'd like to `setError` private because we don't want devs to have to manage the `submitting` flag